nodes/linux/linux.go

```diff
@@ -96,6 +99,47 @@ func (n *linux) PostDeploy(ctx context.Context, _ *nodes.PostDeployParams) error
 		log.Error(err)
 	}
 
+	if n.Cfg.StartupConfig != "" {
+		log.Debugf("Running user-specified startup config for Linux '%s' node", n.Cfg.ShortName)
+
+		startupConfigFilename := path.Base(n.Cfg.StartupConfig)
+
+		// read the startup config file from the host disk.
+		c, err := os.ReadFile(n.Cfg.StartupConfig)
+		if err != nil {
+			return err
+		}
```

Comment:
hi @hawkinsw 

I feel like this feature is not that relevant for the Linux kind. The linux kind is for a generic container, therefore it can not rely on `bash` for instance, to be available.

What you achieve here is realised via generic concepts of `binds` and `exec`. For example, if you want to execute with `bash` a file `my-commands.sh` that you have on disk locally, you would define your node as:

```yaml
mylinuxnode:
  kind: linux
  binds:
    - my-commands.sh:/tmp/my-commands.sh
  exec:
    - bash /tmp/my-commands.sh
```